src/index.ts

```diff
@@ -9,6 +9,7 @@ export {
 	ButtonGroupOption,
 } from './components/ButtonGroup';
 export { Card, CardProps } from './components/Card';
+export { Changelog, ChangelogProps } from './components/Changelog';
```

Comment:
let's export the `processChangelog` function as well so we don't have to do any deep import

Reply:
Resolved here: https://balena.zulipchat.com/#narrow/stream/346007-balena-io.2FbalenaCloud/topic/Changelog.20-.3E.20rendition/near/308960660